addons/medical_gui/stringtable.xml

```diff
@@ -1348,5 +1348,14 @@
         <Key ID="STR_ACE_Medical_GUI_TreatmentItemCount_Vehicle">
             <English>in vehicle's inventory</English>
         </Key>
+        <Key ID="STR_ACE_Medical_GUI_TourniquetWarning">
+            <English>No effect while tourniqeted</English>
```

Comment:
I think phrasing it as `No effect until tourniqet removed` might be better.
You can still perform the action, and it will have an effect, it just gets delayed until after the TQ is taken off.